connectors/byoc.py

```diff
@@ -529,6 +543,12 @@ async def error(self, error):
         self.doc_source["error"] = str(error)
         await self.sync_doc()
 
+    async def _sync_suspended(self, job):
+        await job.suspend()
+        self.doc_source["last_sync_status"] = e2str(job.status)
```

Comment:
Considering updating `last_synced` and `last_synced_error`

Reply:
Good call, thanks for noticing!

Reply:
Fixed in https://github.com/elastic/connectors-python/pull/376/commits/7138f9e292cba3f26de0a2f2c15726ded8322a4b